### PR TITLE
Docs: Move `about` page content

### PR DESCRIPTION
**Pull request by broccolinisoup**
Closes https://github.com/github/primer/issues/1618

### CODE REVIEW

File: src/components/member-container.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react'
+import {Box} from '@primer/react'
+import MemberQuestions from './Member-questions'
```
Comment (colebemis) [inline src/components/member-container.js:3]:
```suggestion
import MemberQuestions from './member-questions'
```

File: src/components/member-container.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import AvatarShape from './Avatar-shape'
```
Comment (colebemis) [inline src/components/member-container.js:4]:
```suggestion
import AvatarShape from './avatar-shape'
```

**Comment by broccolinisoup** [comment]
@emilybrick @colebemis Played with the styling a little bit to fit into the page where we have the nav. Let me know if you have any feedback 🙌🏻

**Review by colebemis — APPROVED** [review]
Looks great, @broccolinisoup!

Made a few tweaks:
- Removed the illustration at the top of the page. I don't think it works well on a light background.
- Made a few layout adjustments to make the page more responsive.
- Fixed the color of links in the answer sections.

**Comment by broccolinisoup** [comment]
> Looks great, @broccolinisoup!
> 
> Made a few tweaks:
> 
> * Removed the illustration at the top of the page. I don't think it works well on a light background.
> * Made a few layout adjustments to make the page more responsive.
> * Fixed the color of links in the answer sections.

Looks great - thanks!

**Pull request merged**